static/src/stylesheets/module/facia/_containers.scss

```diff
@@ -42,6 +42,20 @@
 .fc-container--outbrain {
     background-color: $neutral-8;
     display: none;
+
+    .fc-container__inner {
+        @include mq($from: mobile) {
+            min-height: 621px;
+        }
+
+        @include mq($from: tablet) {
+            min-height: 455.17px;
```

Comment:
I'll add a comment but the reason is that this is the natural height of a fully-loaded (inc images) outbrain widget at that breakpoint - I'm just ensuring that the widget already has that height when it's inserted into the page.